client/src/components/SmallActionButton/SmallActionButton.module.css

```diff
@@ -0,0 +1,11 @@
+.buttonStyle {
+  background: #032a5c;
+  color: white;
+  border-radius: 20px;
+  padding: 3px 10px;
+  margin-top: 10px;
+  border: none;
+
```

Comment:
I would also add in: `cursor: pointer`
This will give a better UI experience with the cursor changing to a pointer

Reply:
Minor comment: Perhaps even adding in a hover style where the background is a darker shade of blue might be good as well

Reply:
Added `cursor: pointer`. Regarding the extra style, I will try pick a shade of blue and at this in later since this isn't too much work. However, I will prioritize getting other fixes done first.

Reply:
Noted.